package.json

```diff
@@ -88,6 +88,7 @@
 	"resolutions": {
 		"**/@automattic/calypso-build/node-sass": "^4.13.1",
 		"**/gulp/gulp-cli/yargs/yargs-parser": "^20.0.0",
+		"**/hosted-git-info": "^2.8.9",
```

Comment:
This sort of thing has been problematic in the past. If something depends on 3.0.8 or 4.0.2, for example, this will still override it to 2.8.9.

If we wind up switching to pnpm, [their version of this feature](https://pnpm.io/package_json#pnpmoverrides) does better in that we can target the override to specific versions of hosted-git-info.

And considering the changes I see in the yarn.lock files in this PR, it shouldn't be needed anyway. The dep is on hosted-git-info@^2.1.4, which is already satisfiable by 2.8.9 so all we need is the update.